templates/gcc_config.bzl.tpl

```diff
@@ -0,0 +1,16 @@
+GCC_BUILTIN_INCLUDE_PATHS = [
+    # Keep sorted
```

Comment:
We could flip the extensions and replace the placeholder with a valid starlark identifier (e.g. `__SOMETHING__`) to get sorting to work

Reply:
I thought about that too, but it would require us to refactor all the references.
Will defer it to a later PR.